src/twisted/trial/test/detests.py

```diff
@@ -208,6 +215,32 @@ def timedOut(err):
 
     test_errorPropagation.timeout = 0.1  # type: ignore[attr-defined]
 
+    async def test_cancelSuppression(self):
+        async def sleep(delay):
+            await task.deferLater(clock=reactor, delay=delay)
+
+        try:
+            await sleep(0.2)
+        except defer.CancelledError:
+            await sleep(0.1)
+
+    test_cancelSuppression.timeout = 0.1  # type: ignore[attr-defined]
+
+    class MyError(Exception):
```

Comment:
This seems like a weird place to put this - any particular reason it's here?

Reply:
So I can check issubclass in the corresponding test case

Reply:
here: https://github.com/twisted/twisted/pull/1399/files#diff-5696cfe643e8b3e6e895831e510f76a90262ecac11a8f5e38e0d68c173d72ecdR253